newrelic.yml

```diff
@@ -8,8 +8,8 @@
 # For full documentation of agent configuration options, please refer to
 # https://docs.newrelic.com/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration
 
-common: &default_settings
-  # Required license key associated with your New Relic account.
+common:
+  &default_settings # Required license key associated with your New Relic account.
```

Comment:
```suggestion
common: &default_settings 
# Required license key associated with your New Relic account.
```

I think `common:` and  `&default_settings` need to be on the same line so the node mapping works. Did this indentation strategy work for you?

Reply:
this ended being a formatter that was operating on save -- it looks like it's valid, but it was unintentional, so moved it back to how it was anyways.